src/group_theory/subgroup.lean

```diff
@@ -1824,82 +1883,109 @@ set.subset.antisymm
     by { rw sup_eq_closure, apply Inf_le _, dsimp, refl })
   ((sup_eq_closure N H).symm ▸ subset_closure)
 
-end subgroup
-
-end pointwise
-
-namespace is_conj
-open subgroup
-
-lemma normal_closure_eq_top_of {N : subgroup G} [hn : N.normal]
-  {g g' : G} {hg : g ∈ N} {hg' : g' ∈ N} (hc : is_conj g g')
-  (ht : normal_closure ({⟨g, hg⟩} : set N) = ⊤) :
-  normal_closure ({⟨g', hg'⟩} : set N) = ⊤ :=
+@[to_additive] lemma mod_law_left (A' A B : subgroup G) (hA : A' ≤ A) :
+  (A' : set G) * (A ⊓ B : subgroup G) = A ⊓ (A' * B) :=
 begin
```

Comment:
Looks like the merge got confused and deleted additions to the `master` branch. Can I push a fixed commit?

Reply:
Sure, sorry about the deletions.

Reply:
Fixed!